pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -64,6 +65,12 @@ func NewDefaultConfigProducerRegistry() *ConfigProducerRegistry {
 			return
 		})
 
+	registry.RegisterPriority(priorities.TaintTolerationPriority,
+		func(args ConfigProducerArgs) (plugins config.Plugins, pluginConfig []config.PluginConfig) {
```

Comment:
A nit: pluginConfig -> pluginConfigs.

(We have multiple places naming like this, so can be addressed in another PR.)

Reply:
ok, lets do a cleanup followup PR.